cmd/root.go

```
@@ -52,9 +52,11 @@ func runRootCmd(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	cachePath, _ := fx.HomeDirPath()
```

This just ignores any errors but I'm pretty sure that was already happening before, just in a different place?

---

Alternatively, `CsvRatesCache` could handle an empty `Path` and just default to `HomeDirPath` to avoid require callers to do this.

---

Yes, the error will show up when you try to read or write the file, but they seem to be handled.